### DISCUSSION TITLE
windows ink doesnt work

### DISCUSSION THREAD

**Discussion by VonnikTheFirst** [OpenTabletDriver/OpenTabletDriver#1228] [Q&A] [answered]
Hello, I just downloaded everything and followed all the steps, and Since I'm using my tablet for both drawing and playing osu! I wanted to get windows ink plugin so I can use pressure for drawing. But the plugin even if its installed it doesnt work nor it get detected in the plugin/filter/interpolars tab. Does anyone have any ideas how to fix this?

**Comment by InfinityGhost** [comment, marked as answer]
The Windows Ink plugin provides an output mode and a tip binding. You can select output modes via the dropdown (likely says "Absolute Mode") and it will show up there.

**Reply by VonnikTheFirst** [reply]
ah thanks! But now my tablet just doesn't work it's like it doesn't detect the pen going over it. and it doesn't output anything.
 (in the video, the first test is always mouse, and then is the tablet)
https://user-images.githubusercontent.com/50554345/123954116-4dec5880-d9a8-11eb-80ef-6eec2d98cb83.mp4

**Reply by InfinityGhost** [reply]
In the video you didn't actually select the right output mode, you need the Windows Ink Absolute Mode to use Windows Ink.

**Reply by VonnikTheFirst** [reply]
I did select the windows ink absolute mode? The second time I switched I switched to windows ink absolute mode.

**Reply by VonnikTheFirst** [reply]
what should I do?

**Reply by InfinityGhost** [reply]
Send the console log here. I can't do any further debugging without it.

**Reply by VonnikTheFirst** [reply]
I just realized I have the same problem as SempaiYeet, since I installed the Vmulti extention. I'll send the new console log.
[New Text Document.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/6777034/New.Text.Document.txt)

**Reply by VonnikTheFirst** [reply]
hello? how is it going?

**Comment by IcedMochaTea** [comment]
i have the same problem and idk how to fix the thing, like absolute mode works but windows ink just doesnt pick up any movement

**Reply by InfinityGhost** [reply]
What does your console log say? Copy it and send it here.

**Comment by SempaiYeet** [comment]
Windows Ink absolute mode picks up movement for me and it picks up when I press down, but it doesn't have any pressure sensitivity

**Reply by InfinityGhost** [reply]
Send the console log.

**Reply by SempaiYeet** [reply]
[Detect:Info]	Searching for tablet 'Gaomon 1060 Pro'
[Detect:Info]	Searching for tablet 'Gaomon M106K Pro'
[Detect:Info]	Searching for tablet 'Gaomon M106K'
[Detect:Info]	Searching for tablet 'Gaomon M10K'
[Detect:Info]	Searching for tablet 'Gaomon S56K'
[Detect:Info]	Searching for tablet 'Gaomon S620'
[Detect:Info]	Searching for tablet 'Huion 1060 Plus'
[Detect:Info]	Searching for tablet 'Huion G10T'
[Detect:Info]	Searching for tablet 'Huion GC610'
[Detect:Info]	Searching for tablet 'Huion H1060P'
[Detect:Info]	Searching for tablet 'Huion H1161'
[Detect:Info]	Searching for tablet 'Huion H430P'
[Detect:Info]	Searching for tablet 'Huion H610 Pro V2'
[Detect:Info]	Searching for tablet 'Huion H610 Pro'
[Detect:Info]	Searching for tablet 'Huion H640P'
[Detect:Info]	Found tablet 'Huion H640P'
[Settings:Info]	Output mode: Windows Ink Absolute Mode
[Settings:Info]	Display area: [2560x1440@<3200, 720>:0°],
[Settings:Info]	Tablet area: [160x90@<79.99999, 50.95238>:0°],
[Settings:Info]	Clipping: Enabled
[Settings:Info]	Ignoring reports outside area: Disabled
[Settings:Info]	Filters: VoiDPlugins.Filter.PrecisionControl, RadialFollow.RadialFollowSmoothing
[Settings:Info]	Tip Binding: [VoiDPlugins.OutputMode.WinInkButtonHandler]@0%
[Settings:Info]	Pen Bindings: [0, VoiDPlugins.OutputMode.WinInkButtonHandler], [1, VoiDPlugins.OutputMode.WinInkButtonHandler]
[Settings:Info]	Express Key Bindings: [0, ], [1, ], [2, ], [3, ], [4, ], [5, ], [6, ], [7, ]
[Settings:Info]	Driver is auto-enabled.

**Reply by InfinityGhost** [reply]
Get the version number of your Windows installation. Might be due to a Windows update breaking VMulti like we've seen recently.

**Reply by VonnikTheFirst** [reply]
19042.1052 - thats my OS build, is this the one braking it?

**Reply by SempaiYeet** [reply]
I got Version 21H1 (OS Build 12043.1083)

**Comment by gonX** [comment]
If you still have issues with Windows Ink please see the answer in #1239
